src/lib/mapbox/styling/index.js

```diff
@@ -1,8 +1,5 @@
 export function decorateLayerStyling(layerData) {
-  if (
-    layerData.properties['deltares:source-layer'] &&
-    layerData.properties['deltares:type'] === 'circle'
```

Comment:
@florislangeraert I removed this test for type circle as the one you were testing was type symbol. Either way it should grab the paint styling directly over the API then.